tracks/dart/exercises/two-fer/mentoring.md

```diff
@@ -0,0 +1,56 @@
+### Reasonable solutions
+
+The most simple solution is to set the default value for the parameter to `"you"` and use string interpolation:
+
+```dart
+String twofer([String name = "you"]) {
+  return "One for $name, one for me.";
+}
+```
+
+As an alternative, one could use an expression-bodied method:
+
+```dart
+String twofer([String name = "you"]) => "One for $name, one for me.";
+```
+
+An alternative solution would be to use set the default value to `null` and use the 
+[null-coalescing operator][reference-null-coalescing-operator]:
+
+```dart
+String twofer([String name = null]) {
+  return "One for ${name ?? "you"}, one for me.";
```

Comment:
I suggest removing this entire solution altogether, as there is a common suggestion to set it to "you".